cmd/kube-scheduler/app/server.go

```diff
@@ -143,7 +144,7 @@ func runCommand(cmd *cobra.Command, args []string, opts *options.Options) error
 
 	// Apply algorithms based on feature gates.
 	// TODO: make configurable?
-	algorithmprovider.ApplyFeatureGates()
+	schedulerFeatureGateDependencies := algorithmprovider.ApplyFeatureGates()
```

Comment:
Can we call this function `Run` directly or is there dependency in blocks further down in the function?

Reply:
done